MonoXAnalysis/python/postprocessing/examples/genFriendProducer.py

```diff
@@ -200,5 +209,6 @@ def analyze(self, event):
 
 # define modules using the syntax 'name = lambda : constructor' to avoid having them loaded when not needed
 
-genQEDJets = lambda : GenQEDJetProducer(deltaR=0.1)
+genQEDJets = lambda : GenQEDJetProducer(deltaR=0.1,beamEn=7000.)
```

Comment:
I think these should be 6500 for sqrt(s)=13 TeV (there was a bug before with beam energy = 7 TeV coming from 14 TeV LHC) and eventually 4000 for 8 TeV

Reply:
OK, my misunderstanding, 7000 was the value that was there and for some reason my brain went to sqrts=7 TeV